detectron2/modeling/mmdet_wrapper.py

```diff
@@ -145,6 +145,7 @@ def __init__(
 
             detector = build_detector(_to_container(detector))
         self.detector = detector
+        self.detector.init_weights()
```

Comment:
Is this needed? IIRC weights can be initialized the same way as other detectron2 models through the checkpointer.

Reply:
This is to call random weight initialization methods, not to load checkpoints: https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/base_module.py#L56

Reply:
Nice! thanks for the clarification.